bots/images/scripts/rhel-7-4.setup

```diff
@@ -82,8 +82,6 @@ NetworkManager-team \
 PackageKit \
 pcp-libs \
 pcp \
-python-pip \
```

Comment:
if the rhel-7-4 setup is changed, I think we should also build a new image and test that before merging